definitions/contested/json/CaseEvent/CaseEvent-consentedInContested-prod.json

```diff
@@ -0,0 +1,54 @@
+[
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "FinancialRemedyContested",
+    "ID": "FR_uploadConsentedOrderNoPayment",
+    "Name": "Consent Order (No Payment)",
+    "Description": "Consent Order (No Payment)",
+    "DisplayOrder": 24,
+    "PreConditionState(s)": "*",
+    "PostConditionState": "*",
+    "SecurityClassification": "Public",
+    "ShowSummary": "N",
+    "ShowEventNotes": "Y"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "FinancialRemedyContested",
+    "ID": "FR_uploadConsentedOrderNoPayment_solicitor",
+    "Name": "Consent Order (No Payment)",
+    "Description": "Consent Order (No Payment)",
+    "DisplayOrder": 23,
+    "PreConditionState(s)": "caseAdded;prepareForHearing;solicitorDraftOrder",
+    "PostConditionState": "*",
+    "SecurityClassification": "Public",
+    "ShowSummary": "N",
+    "ShowEventNotes": "Y"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "FinancialRemedyContested",
+    "ID": "FR_uploadConsentedOrderWithPayment",
+    "Name": "Consent Order Payment",
+    "Description": "Consent Order Payment",
+    "DisplayOrder": 23,
+    "PreConditionState(s)": "*",
```

Comment:
I have a feeling that removing this event will prevent the current cases in prod from being able to complete the existing consent flow in PROD assuming there will still be cases when this is released

Reply:
Might be correct - will confirm with Grace what the expected behaviour of this is

Reply:
So having spoken to Grace - there is only 3 cases in Production that are possibly affected by this, and Lucy believes they may be old cases anyway and they're happy to handle these manually if necessary